common/annotationsDerived/af.xml

```diff
@@ -462,6 +462,86 @@ CLDR data files are interpreted according to the LDML specification (http://unic
 		<annotation cp="👱🏾" type="tts">persoon met blonde hare: medium-donker velkleur</annotation>
 		<annotation cp="👱🏿">blond | donker velkleur | hare | persoon met blonde hare</annotation>
 		<annotation cp="👱🏿" type="tts">persoon met blonde hare: donker velkleur</annotation>
+		<annotation cp="🧑🏻‍❤‍💋‍🧑🏼">ligte velkleur | medium-ligte velkleur | paartjie | romanse | soen | volwassene</annotation>
```

Comment:
I think these new ones happened as a result of the updates for Emoji 13.1. My fault for not rerunning; glad you did this.